scripts/run_locally_with_docker.sh

```diff
@@ -15,7 +15,7 @@ API_HOST_NAME=${API_HOST_NAME:-"http://host.docker.internal:6011"}
 # Only expose port 6011 if we're running the API - anything else is celery which shouldn't bind the port.
 # This lets us run celery via docker and the API locally.
 if [ "${@}" == "api" ]; then
-  EXPOSED_PORTS="-p 6011:6011"
+  EXPOSED_PORTS="-e PORT=6011 -p 6011:6011"
```

Comment:
I don't think this needs any change but I know we were discussing the other time how the PaaS version of this is configured.

I did a search in the api repo and the aws repo for `PORT` and 6011 but couldn't find anything relevant.

I couldn't figure out why. However, I've ssh'd onto one of the preview instances of `notify-api` and `notify-delivery-worker-sender` and then run `echo $PORT` which returns `8080`.

My guess is that this environment variable is set by the PaaS team automatically for us on the PaaS (and is value 8080 whereas on local we specify 6011, for reasons I don't remember).

So my thinking is that
- we require an environment variable for PORT when running gunicorn because our config file requires it (https://github.com/alphagov/notifications-api/blob/main/gunicorn_config.py#L14)
- runnning gunicorn on the PaaS we don't need to set a PORT because the PaaS set the environment variable for us and that is why we can't find any reference to it in our code
- running flask locally we don't actually need to set a port because flask has a default port that it exposes flask on (probably 8080), nor does it use the PORT environment variable anywhere in our python app. However we choose when running flask locally to override the default port flask is exposed on by calling `flask run -p 6011' in our makefile
- if we were to run gunicorn locally or via docker, either way we need to set the PORT environment variable for it to work

Reply:
cloudfoundry does indeed expect the apps to run on 8080: https://docs.cloudfoundry.org/buildpacks/python/index.html#run-web-server.

When running things locally we need to have the apps listening on different ports so that they don't overlap.

The question I guess is, do we want to have the same ports used when we deploy things to AWS or we want to use a more homogenous approach where all the http apps listen on 8080?

I think I would prefer the later as it makes it simpler to reason about what's listening on which port and to have one less thing to keep in your mind, but then again developers may already know these ports by heart.